providers/common/sql/pyproject.toml

```
@@ -68,6 +68,9 @@ dependencies = [
 # The optional dependencies should be modified in place in the generated file
 # Any change in the dependencies is preserved when the file is regenerated
 [project.optional-dependencies]
+"common.compat" = [
+    "apache-airflow-providers-common-compat>=1.7.4",  # + TODO: bump to next version
```

In parallel this has been bumped to 1.8.0 already
```suggestion
    "apache-airflow-providers-common-compat>=1.8.0",
```